src/pages/trips/plan.tsx

```diff
@@ -41,41 +53,52 @@ const Plan = () => {
+                <label>Where are you going? Please enter multiple destinations as comma-separated values. </label>
+                <input onChange={handleDestinationChange} type='text' className='w-full p-1 rounded-md outline-none text-black'/>
             </div>
 
             {/* Input for dates (calendar) */}
             <p className='mt-5'>Dates</p>
-            <div className='bg-white p-3 w-full rounded-md flex'>
-                <p>{value[0]!.toLocaleDateString()} -</p>
-                <p>- {value[1]!.toLocaleDateString()}</p>
+            <div className='bg-white p-3 w-full rounded-md flex text-black'>
+                <p>{value[0]!.toLocaleDateString()} </p>
+                &nbsp;
+                <p>-</p>
+                &nbsp;
+                <p>{value[1]!.toLocaleDateString()}</p>
             </div>
             <div className='mt-5 w-full'>
-                <Calendar onChange={onChange} value={[value[0]!, value[1]!]} goToRangeStartOnSelect={false} selectRange={true} calendarType='US' minDate={new Date()} className='m-auto'/>
+                <Calendar 
+                    onChange={onChange} 
+                    value={[value[0]!, value[1]!]} 
+                    goToRangeStartOnSelect={false} 
+                    selectRange={true} 
+                    calendarType='US' 
+                    minDate={new Date()} 
+                    className='m-auto text-black'
+                />
             </div>
 
             <div className='mt-5'>
                 <label>Invite anyone going with you (optional)</label>
-                <input type='text' className='w-full p-1 rounded-md outline-none'/>
+                <input type='text' className='w-full p-1 rounded-md outline-none text-black'/>
                 <p className='text-sm italic mt-1 text-slate-300'>Write emails seperated by a comma and a space</p>
             </div>
             
             <div className='mt-5'>
                 <label>Would you like your itinerary to be discoverable by other voyagers?</label>
                 <select className='block p-1 rounded-md w-full'>
-                    <option>📢 Public</option>
-                    <option>🔒 Private</option>
+                    <option onChange={() => setIsPublic(true)}>📢 Public</option>
+                    <option onChange={() => setIsPublic(false)}>🔒 Private</option>
                 </select>
                 <p className='text-slate-300 text-sm italic mt-1'><AiOutlineInfoCircle size={15} className='inline-block mr-1'/>For your privacy, this will only happen once your trip is over</p>
             </div>
             
             <div className='flex justify-center mt-5'>
-                <button onClick={getItineraryByID} className='bg-cyan-400 py-2 px-8 rounded-lg text-slate-50 hover:bg-cyan-500'>Create trip</button>
+                <button onClick={handleClick} className='bg-cyan-400 py-2 px-8 rounded-lg text-slate-50 hover:bg-cyan-500'>Create trip</button>
```

Comment:
We can either remove the preventDefault in the handler or make this a button of type submit. The latter case would make more sense since this page is a form basically. We can do this by wrapping all elements that have input fields in form tags. If the button is not wrapped in the form tags, we can add an id for the form element and a formEvent attribute (I believe its called) on the button with the value of the id on the form.